src/proto/Polkadot.proto

```diff
@@ -4,10 +4,10 @@ package TW.Polkadot.Proto;
 option java_package = "wallet.core.jni.proto";
 
 // Known networks
-enum Network {
+/*enum Network {
     POLKADOT = 0;
     KUSAMA = 2;
-}
+}*/
```

Comment:
to remove later

Reply:
or we can keep it for compatibility? Then cast enum raw value to `uint32`

Reply:
I think it's better to remove it and mention it to the release, it's an easy upgrade.